docs/installation-guide/src/main/asciidoc/installing.adoc

```diff
@@ -660,49 +565,49 @@ Table 1-2 {productName} Full Profile and Web Profile Features
 |===
 |Feature |Full Platform |Web Profile
 
-|Java Platform, Enterprise Edition 8 |X |X
+|Jakarta EE 10 Platform |X |X
 
-|Jakarta EE Security API |X |X
+|Jakarta Security 3.0 supporting OpenID Connect |X |X
 
-|Java API for JSON Processing 1.1 |X |X
+|Jakarta JSON Processing 2.1.3 |X |X
 
-|Java API for JSON Binding 1.0 |X |X
+|Jakarta JSON Binding (JSON-B) 3.0.1 |X |X
 
-|EJB 3.2 Lite (Support for Local Container only) |X |X
+|EJB 4.0.1 Lite (Support for Local Container only) |X |X
 
 |JTA 1.2 |X |X
 
 |OSGi-based microkernel |X |X
 
-|Java Web Technologies (Servlet 4.0, JSP 2.2, JSF 2.3, JSTL 1.2, EL 3.0) |X |X
+|Java Web Technologies (Servlet 6.0, JSP 3.1.1, JSF 4.0.4, JSTL 3.0.2, EL 5.0) |X |X
 
 |Grizzly/Comet |X |X
 
-|JPA 2.2, (EclipseLink), JDBC Connection Pooling |X |X
+|JPA 4.0.5, (EclipseLink), JDBC Connection Pooling |X |X
 
 |JDBC 4.0 |X |X
 
-|Apache Derby 10.13.1.1 |X |X
+|Apache Derby 10.15.2.0 |X |X
```

Comment:
Derby is not a Jakarta EE API, we are not bound to any version - we can upgrade it to any version, it's not worth keeping the version number in this guide. Unless we generate the version from pom.xml, which would be nice, but we don't do it now. It's better to remove the version rather than keep an outdated version.
```suggestion
|Apache Derby|X |X
```

Reply:
Actually it could be worth to have the concrete version and also to link to its documentation from our documentation, but that is probably work for another task. However ... when you mentioned filtering, we have these properties in POMs.